README.md

```diff
@@ -34,6 +34,7 @@ The JBeret Quarkus extension supports the following configuration:
  |---|---|---|
  | `quarkus.jberet.repository`<br>The repository type to store JBeret and Job data. A `jdbc` type requires a JDBC datasource. | `in-memory`, `jdbc`  | `in-memory` |
  | `quarkus.jberet.repository.jdbc.datasource`<br>The datasource name. | string  | `<default>` |
+ | `quarkus.jberet.max_processors`<br>A parameter to specify maximum processor count.  | int  | |
```

Comment:
I think it will be better if we use quarkus global configuration for the thread pool:
https://github.com/quarkusio/quarkus/blob/main/core/runtime/src/main/java/io/quarkus/runtime/ExecutorRecorder.java#L157

Because if we add a new configuration and both values are not related at all, we can fall into other issues (bigger permits in JBeret than the actual thread pool size). 

My proposal is to just drop the current implementation and replace it with how Quarkus determines the thread pool size, so it is consistent.

Reply:
Got it and changes are ready for you to review now. Thanks!!